lib/controlling/RouterBrewery.js

```diff
@@ -26,59 +22,11 @@ var RouterBrewery = {
 };
 
 function createRouter(props, classProps) {
-    var RouterType = Environment.isServer() ? ServerRouter : Router;
-    var BrisketRouter = RouterType.extend(augmentRouteHandlers(props), classProps);
-
-    BrisketRouter.extend = throwsNoExtendAllowed;
+    var BrisketRouter = Router.extend(props, classProps);
 
     return BrisketRouter;
 }
 
-function throwsNoExtendAllowed() {
```

Comment:
now that the router brewery no longer modifies the route handlers, it is safe to bring back Router.extend